plugin.php

```diff
@@ -237,19 +233,20 @@ function sailthru_save_post( $post_id, $post, $post_before ) {
 			$client     = new WP_Sailthru_Client( $api_key, $api_secret );
 			try {
 				if ( $client ) {
-					$data = array();
+					$site_metadata = get_option( 'vl_metadata' );
+
+					$data = [];
 					// Prepare the Content API Params
 					$data['url']               = get_permalink( $post->ID );
 					$data['title']             = $post->post_title;
 					$data['author']            = get_the_author_meta( 'display_name', $post->post_author );
 					$data['date']              = $post->post_date;
 					$data['vars']['post_type'] = $post->post_type;
+					$data['vars']['postID']    = $post->ID;
+					$data['vars']['siteID']    = $site_metadata['id']; // TBD: site id field
```

Comment:
@andremetzen I'm not sure what are we naming the site id inside the metadata.

Reply:
I guess we need to start drafting a metadata schema. I would start using something like `site_id` instead of just `id`